src/cloudai/workloads/megatron_run/megatron_run.py

```
@@ -81,8 +81,16 @@ def no_dashed_args(self):
     @property
     def cmd_args(self) -> dict[str, Union[str, list[str]]]:
         args = self.model_dump(exclude_none=True, exclude={"docker_image_url", "run_script"})
-        args = {f"--{k.replace('_', '-')}": v for k, v in args.items()}
-        return args
+        result: dict[str, Union[str, list[str]]] = {}
```

Ruff (0.15.21)

[warning] Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)